src/app/pages/maps/maps.module.ts

```diff
@@ -5,18 +5,26 @@ import { AngularEchartsModule } from 'ngx-echarts';
 
 import { ThemeModule } from '../../@theme/theme.module';
 import { MapsRoutingModule, routedComponents } from './maps-routing.module';
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
 
 @NgModule({
   imports: [
     ThemeModule,
-    AgmCoreModule.forRoot(),
+    AgmCoreModule.forRoot({
+      apiKey: 'AIzaSyCpVhQiwAllg1RAFaxMWSpQruuGARy0Y1k',
+      libraries: ["places"]
+    }),
     LeafletModule.forRoot(),
     MapsRoutingModule,
     AngularEchartsModule,
+    FormsModule,
+    ReactiveFormsModule
   ],
   exports: [],
   declarations: [
-    ...routedComponents,
```

Comment:
looks like auto formatting changes - let's avoid those as it usually makes the changelist messy